src/Templates/VS2017/Roslyn.SDK.VS2017.csproj

```diff
@@ -11,6 +11,7 @@
     <!-- VS Insertion -->
     <VisualStudioInsertionComponent>Microsoft.CodeAnalysis.SDK</VisualStudioInsertionComponent>
     <Ngen>false</Ngen>
+    <UseCodeBase>true</UseCodeBase>
```

Comment:
❓ Why is this needed with `<GeneratePkgDefFile>false</GeneratePkgDefFile>`?

Reply:
@tmat 	for the answer...

Reply:
Well I'll take a stab at it.  We don't generate a pkgdef for this project but we do for its dependencies.  I'll do some testing, perhaps this is not required on this project.